velox/functions/sparksql/Register.cpp

```diff
@@ -232,7 +232,10 @@ void registerFunctions(const std::string& prefix) {
       Varchar>({prefix + "unix_timestamp", prefix + "to_unix_timestamp"});
   registerFunction<MakeDateFunction, Date, int32_t, int32_t, int32_t>(
       {prefix + "make_date"});
-
+  registerFunction<DateDiffFunction, int32_t, Date, Date>(
+      {prefix + "date_diff"});
+  registerFunction<DateDiffFunction, int32_t, Date, Date>(
+      {prefix + "datediff"});
```

Comment:
Make sure to document both functions.

Reply:
As Spark only has `datediff`, I just removed `date_diff`.